darwin/dataset/download_manager.py

```diff
@@ -95,11 +95,13 @@ def download_all_images_from_annotations(
 
         if not force_replace:
             # Check the planned path for the image against the existing images
-            planned_image_path = (
-                images_path
-                / Path(annotation.remote_path.lstrip("/\\")).resolve().absolute()
-                / Path(annotation.filename)
-            )
+            filename = annotation.filename
+            if use_folders and annotation.remote_path != "/":
+                planned_image_path = (
+                    images_path / Path(annotation.remote_path.lstrip("/\\"))
+                ) / filename
+            else:
+                planned_image_path = images_path / filename
```

Comment:
This allows `planned_image_path` to work with dataset releases pulled in a flat structure